### PR TITLE
solidity docs for mock methods

### LINKED ISSUE #7: add Solidity "natural" docs to the library
https://docs.soliditylang.org/en/develop/natspec-format.html

### PR DESCRIPTION
**Pull request by raulk**
Closes https://github.com/Zondax/fevm-solidity-mock-api/issues/7.

### CODE REVIEW

File: contracts/v0.8/MinerAPI.sol [hunk 1]
```diff
@@ -23,11 +23,15 @@ contract MinerAPI {
         sectorSizesBytes[CommonTypes.SectorSize._64GiB] = 2 * (32 << 30);
     }
 
+    /// (Mock method) Sets the owner of a Miner, which will be returned via get_owner().
     function mock_set_owner(string memory addr) public {
         require(bytes(owner).length == 0);
         owner = addr;
     }
 
+    /// Returns the owner address of a Miner.
```
Comment (emmanuelm41) [inline contracts/v0.8/MinerAPI.sol:32]: Is this correct? should we use @return as it says here? 
https://docs.soliditylang.org/en/develop/natspec-format.html

Comment (raulk) [reply contracts/v0.8/MinerAPI.sol:32]: This is method docs, this is what the method does. But I agree we should document return types via `@return` tags too! Can we submit this patch and maybe you can submit a follow-up patch with the `@return` texts?

**Review by emmanuelm41 — APPROVED** [review]
It is ok for now. I will merge it and then fix styling on comments.

File: contracts/v0.8/MinerAPI.sol (hunk 1 above)
Comment (emmanuelm41) [reply contracts/v0.8/MinerAPI.sol:32]: Done!

**Pull request merged**